src/RazorSdk/SourceGenerators/RazorSourceGenerationOptions.cs

```diff
@@ -35,5 +36,10 @@ internal class RazorSourceGenerationOptions
         /// </para>
         /// </summary>
         public bool SuppressRazorSourceGenerator { get; set; } = false;
+
+        /// <summary>
+        /// Gets the CSharp language version currently used by the compilation.
+        /// </summary>
```

Comment:
LangVersion.10? Latest preview is a bad default to pick no?

Reply:
Generally, yes. I only set it as the default here since that's what the incremental RSG depends on. Follow-up to undo this is at: https://github.com/dotnet/aspnetcore/issues/33848